### ISSUE TITLE
Double dialog when opening a window for non-trusted domains

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#122323]
Testing #121763

The first dialog should be enough?

<img width="528" alt="Screen Shot 2021-04-27 at 5 38 32 AM" src="https://user-images.githubusercontent.com/2193314/116242523-d9343e00-a71a-11eb-94a4-55204ad18940.png">
<img width="530" alt="Screen Shot 2021-04-27 at 5 38 35 AM" src="https://user-images.githubusercontent.com/2193314/116242530-da656b00-a71a-11eb-8ac5-24b61ed47cf9.png">

**Comment by isidorn** [comment]
Same as https://github.com/microsoft/vscode/issues/122324
If the browser blocked it, then we show the dialog. So it is not enough.
As for how the external website handles the open button, maybe @JacksonKearl can shed some light since I think he owns this

**Comment by JacksonKearl** [comment]
Assuming you clicked "open" in the first dialog? That would result in a call to each of your registered external openers to see if they support opening the link, and only after all those async calls finish we try to open it with the default opener. If the async calls take too long the browser will give the warning. This does seem to be the same as #122324, cc @mjbvz / @jrieken .

**Comment by isidorn** [comment]
So closing this as duplicate of that issue
@mjbvz @JacksonKearl  please let me know if you would like me to keep this issue open so we have this tracking also in the VS Code repository
